backend/user/management/commands/clearcache.py

```diff
@@ -0,0 +1,38 @@
+import logging
```

Comment:
I'm not too sure of any cases where we'll need to clear cache (first time using caching tbh), but since we are caching, I think it's fine to keep the file, just in case we need to but we don't want to mess with celery. Charley pointed the file out to me when recommending we set up caching.

Reply:
Ask charley when this is used